Assets/HoloToolkit/ARCA/Plugins/WSA/x86/opencv_core310.dll.meta

```diff
@@ -0,0 +1,34 @@
+fileFormatVersion: 2
+guid: c2016eeba9435a94c8da33885b5abb1c
```

Comment:
I'm not sure it's a good idea to include all these dependencies in the toolkit. It's going to introduce issues with others who are already using this plugin.

Reply:
It is not appropriate to include the dependencies. For one, they are licensed differently from MRTK.  Users of ARCA are going to need to install the dependencies on their development PC.


Please *remove* the OpenCV binaries from this PR.

Thanks!